tests/impl.h

```diff
@@ -138,6 +138,7 @@
     TYPE(mm_loadu_pd)                 \
     TYPE(mm_loadu_si128)              \
     TYPE(mm_madd_epi16)               \
+    TYPE(mm_max_epi8)                 \
```

Comment:
`_mm_max_epi8` came with SSE4.1 and you shall move the item into the classified area.